ledger/eval_test.go

```diff
@@ -428,6 +432,36 @@ func TestPrepareEvalParams(t *testing.T) {
 			}
 		})
 	}
+
+	// Test cost pooling on old and new consensus protos
+	for i, testCase := range cases {
+		t.Run(fmt.Sprintf("i=%d", i), func(t *testing.T) {
```

Comment:
this is not must but something you might to consider: there are two almost identical loops.
If fact, it can be done as a single loop over `cases`.
1. Remove fake proto from `eval` construction above
2. Make `params` array consisting on 3 elements: fake params with `Application: true, MaxAppProgramCost: 700`, vFuture, v29.
3. Modify `evalTestCase` struct and add number of app calls in tx group (`numAppCalls`)
4. Run loops over cases, params, and calculate pooled cost as `proto.MaxAppProgramCost * numAppCalls`, and compare against a value EvalParams

Reply:
Good suggestion, will refactor this part.